unit_test/main_options_test.py

```diff
@@ -204,3 +204,18 @@ def test_build_verbosity(build_verbosity, platform_specific, platform, intercept
 
     expected_verbosity = max(-3, min(3, int(build_verbosity or 0)))
     assert intercepted_build_args.kwargs['build_verbosity'] == expected_verbosity
+
+
+@pytest.mark.parametrize('before_test', ["", 'before --test'])
+@pytest.mark.parametrize('platform_specific', [False, True])
+def test_before_test(before_test, platform_specific, platform, intercepted_build_args, monkeypatch):
+    if before_test is not None:
+        if platform_specific:
+            monkeypatch.setenv('CIBW_BEFORE_TEST_' + platform.upper(), before_test)
+            monkeypatch.setenv('CIBW_BEFORE_TEST', 'overwritten')
+        else:
+            monkeypatch.setenv('CIBW_BEFORE_TEST', before_test)
+
+    main()
+
+    assert intercepted_build_args.kwargs['before_test'] == before_test
```

Comment:
While we're at the level of details. Maybe an extra newline, here?
And ... I don't know; do the tests follow the order of arguments in `build`, or not? If so, I'd say move this up a bit; otherwise not necessary.

Reply:
done